### ISSUE TITLE
[cli-web] Remove dependency of "@bluebase/babel-preset-bluebase" package

### ISSUE THREAD

**Issue by artalat** [BlueBaseJS/cli#71]
If a project doesn't have `@bluebase/babel-preset-bluebase` as a dependency, BlueBase CLI web throws following error when trying to build:

```
Error: Cannot find module '@bluebase/babel-preset-bluebase' from ...
````

**Comment by artalat** [comment]
Closed by #73

**Comment by danial24** [comment]
@artalat 
Generated a PR for this issue
https://github.com/BlueBaseJS/cli/pull/75

Please close this as well once the PR gets merged.
